src/integration/Sites.spec.ts

```diff
@@ -156,6 +156,10 @@ describe("Sites Router", () => {
 
   describe("/", () => {
     beforeAll(async () => {
+      await User.sync({ force: true })
```

Comment:
just to clarify, I thought we prevented the DB clash by removing concurrency using the flag 
`--runInBand`? if this solves the concurrency issue, should we remove the flag `--runInBand` to allow for concurrency again?

Reply:
actually i'm not too very sure on the specifics (cc @dcshzj for clarification); my understanding is that `--runInBand` serializes it and we have `.sync` to prevent db pollution between tests

Reply:
okie